pkg/metrics/otel.go

```diff
@@ -106,6 +111,7 @@ processors:
         match_type: strict
         metric_names:
           - rg_reconcile_duration_seconds
+          - kcc_resource_count_total
```

Comment:
Do you mean `kcc_resource_group_total`?

Reply:
This one is accidentally included by the above `regex include` filer with the name `kcc_resource_count.*`. Excluding this as `kcc_resource_count_total` is not defined in `kubernetes.io/` metrics definition. 

This metric has been removed from Resource Group Controller, we can erase this rule once the RGC is updated in Config Sync.

Reply:
Sounds good. 
Can you add a comment regarding this TODO item?

Reply:
Done!